setup-google-cloud-sdk/src/index.ts

```diff
@@ -164,9 +164,10 @@ export function parseServiceAccountKey(
       "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/service-account-email"
     }
     `;
+    let errorMsg = (error instanceof Error) ? error.message : error as string;
```

Comment:
Could we make this a function that takes in an unknown type and returns an Error? That way we can reuse it in other places.

Reply:
done